python/sglang/bench_serving.py

```diff
@@ -637,17 +753,35 @@ def fire(args: argparse.Namespace):
     else:
         raise ValueError(f"Unknown dataset: {args.dataset_name}")
 
-    asyncio.run(
-        benchmark(
-            backend=backend,
-            api_url=api_url,
-            model_id=model_id,
-            tokenizer=tokenizer,
-            input_requests=input_requests,
-            request_rate=args.request_rate,
-            disable_tqdm=args.disable_tqdm,
+    if args.multi:
```

Comment:
Reuse `bench serving` instead of writing a separate `bench rps`, finding a lot of similar code, including **user-specified parameters**.